Bottom: 3b467996941c0e899414bd67f1e28891596173cc
Top:    2bc212a9e1b379a0b34ce5bd9d4ae4d9c8bc61c8
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-02-04 15:57:59 +0000

Adjust circular approximation


---

diff --git a/src/polygon.h b/src/polygon.h
index 369efea..2d65302 100644
--- a/src/polygon.h
+++ b/src/polygon.h
@@ -34,7 +34,7 @@
 
 /* Implementation constants */
 
-#define POLY_CIRC_SEGS 40
+#define POLY_CIRC_SEGS 8
 #define POLY_CIRC_SEGS_F ((float)POLY_CIRC_SEGS)
 
 #if 0
